### PR TITLE
Harmony 1132 - job status link for unpausing

### PR DESCRIPTION
**Pull request by vinnyinverso**
## Jira Issue ID
HARMONY-1132

## Description
Updates the job status page with a link for resuming.

## Local Test Steps
Pause a running job. Navigate to the job status page and take note of the message as well as a link for resuming the job.

## PR Acceptance Checklist
* [x] Acceptance criteria met
* [x] Tests added/updated (if needed) and passing

### CODE REVIEW

**Review by indiejames — APPROVED** [review]
Worked for me when tested locally

File: app/frontends/jobs.ts [hunk 1]
```diff
@@ -47,6 +47,13 @@ function getLinksForDisplay(job: Job, urlRoot: string, statusLinkRel: string): J
   if (job.status === JobStatus.SUCCESSFUL && needsStacLink(dataLinks)) {
     links.unshift(new JobLink(getStacCatalogLink(urlRoot, job.jobID)));
   }
+  if (job.status === JobStatus.PAUSED) {
+    links.unshift(new JobLink({
+      title: 'Resumes the job.',
+      href: `${urlRoot}/jobs/${job.jobID}/resume`,
+      type: 'application/json',
```
Comment (chris-durbin) [inline app/frontends/jobs.ts:54]: We try to have a `rel` attribute on most of the links - maybe call this one `rel: resume` so it is is easier to programmatically find the resume link.

https://www.iana.org/assignments/link-relations/link-relations.xhtml has a list of official ones, but I didn't find anything good in there with a quick check.

**Review by chris-durbin — APPROVED** [review]
Recommend adding a `rel` for the link.

File: app/frontends/jobs.ts (hunk 1 above)
Comment (indiejames) [reply app/frontends/jobs.ts:54]: It sounds stupid, but I think `rel: resumer` would be more in line with the way the relationships are named

**Pull request merged**